docs/Doxyfile

```diff
@@ -13,7 +13,7 @@ EXCLUDE_PATTERNS       = */private/*
 CREATE_SUBDIRS         = NO
 FULL_PATH_NAMES        = YES
 RECURSIVE              = YES
-GENERATE_HTML          = YES
```

Comment:
Is this a big change?

Reply:
No we never used that so I just disabled it. Sphinx uses the xml output